cli/plugin/src/main/java/io/helidon/build/cli/plugin/UpdateMetadata.java

```diff
@@ -185,12 +187,35 @@ private void updateLatestVersion() throws Exception {
                                                           .connectTimeout(connectTimeout)
                                                           .readTimeout(readTimeout)
                                                           .connect();
-        Files.copy(connection.getInputStream(), latestVersionFile, REPLACE_EXISTING);
+        if (connection instanceof HttpURLConnection && ((HttpURLConnection) connection).getResponseCode() == 404) {
```

Comment:
This is weird. Can we just fallback to the old mechanism if `versions.xml` is not found ?

Reply:
Old mechanism is using `latest` file. I removed it from the code. This lines is temporary solution while `version.xml` is not implemented on the web site and it should be removed right after that. Do I need to return `latest` file back?

Reply:
I don't like having a temporary solution. How about publishing a `version.xml` right now instead ?

Reply:
Let`s do it.
Could you please ping me when the file will be published? I have some failing tests because of the missing `version.xml` and I need to recheck these tests.